Util/Schema/whichtype.m

```diff
@@ -0,0 +1,11 @@
+function [argtype] = whichtype(arg),
```

Comment:
Do we need this?

Reply:
Yes - It's an good alias and to define a string instead of getting the 5th argument of objinfo. It what you would use to get a string representation of a type.

Reply:
I mean, again it's a one liner in a separate .m file...

Reply:
That's a feature!

Would you prefer reading:

```[~,~,~,~,type] = objinfo(a)```

instead of 

```type = whichtype(a)```

?

Note also that objinfo uses objinfo (one level recursion). You didn't notice because ```whichtype``` clarify the code intent, otherwise we would have  ```[~,~,~,~,otype{k}] = objinfo(data)``` calls instead.

To delete ```whichtype``` is also to sacrifice expressiveness for code/scope size. For example, everytime you want a type you have to inspect objinfo help to remember it's the 5th output argument, instead of just using a simple function that actually output only what you want.

Being a oneliner is a feature: I'm reusing the code in objinfo to create a function that is a simplified version not available anywhere, with a proper name that matches intent.